database/factories/InviteKeyFactory.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\Game;
+use App\Models\InviteKey;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Illuminate\Support\Str;
+
+class InviteKeyFactory extends Factory
+{
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = InviteKey::class;
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        return [
+            'value'   => Str::random(4),
```

Comment:
Zijn dit hoofdletters? Zitten hier ook nummer tussen? Kun je duplicates krijgen (voor een game)?

Reply:
Dat kan inderdaad gebeuren, maar deze `facotry` is vooral bedoeld voor het testen. De functionaliteit om echt codes te generen gebeurt hier niet.

Reply:
Nee dat snap ik wel, maar het lijkt me wel dat het handig is om onze validation rules te enforcen voor testdata (zodat daar later geen geknoei mee kan komen)

Reply:
Dat is inderdaad wel slim, zal morgen verwerken